orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationModuleConfig.java

```diff
@@ -48,4 +49,16 @@ public class OrchestrationModuleConfig {
 
   /** A content filter to filter the prompt. */
   @Nullable FilteringModuleConfig filteringConfig;
+
+  /**
+   * Creates a new configuration with the given LLM configuration.
+   *
+   * @param aiModel The LLM configuration to use.
+   * @return A new configuration with the given LLM configuration.
+   */
+  @Nonnull
+  public OrchestrationModuleConfig withLlmConfig(@Nonnull final OrchestrationAiModel aiModel) {
+    return new OrchestrationModuleConfig(
+        aiModel.createConfig(), templateConfig, maskingConfig, filteringConfig);
```

Comment:
```suggestion
    WithLlmConfig(
        aiModel.createConfig(), templateConfig, maskingConfig, filteringConfig);
```

otherwise we have to update this when adding more orchestration modules.